.github/update.sh

```diff
@@ -1,20 +1,22 @@
-#!/usr/bin/bash
+#! /usr/bin/env bash
 
-# Updates QT Translation sources, converts them into xlf
+# Updates QT Translation sources, converts them into XLIFF
 # Usage ./import.sh /path/to/mozVPN/ /path/to/translateGit
-# 
+#
 # lupdate & lconvert with qml support must be available
 
+LUPDATE=$(which lupdate)
+LCONVERT=$(which lconvert)
 
-/usr/lib/qt5/bin/lupdate vpn/src/src.pro -ts
+LUPDATE vpn/src/src.pro -ts
```

Comment:
I definitely need to fix these command calls, this should be `$LUPDATE`. Trying to figure out what else causes the difference.